firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1569,9 +1569,9 @@ extension String {
                 value: "Be careful. Something doesn’t look right.",
                 comment: "On error page, this is the title for generic error.")
             public static let Description = MZLocalizedString(
-                key: "NativeErrorPage.GenericError.Description.v131",
+                key: "NativeErrorPage.GenericError.Description.v134",
                 tableName: "NativeErrorPage",
-                value: "An SSL error has occurred and a secure connection to the server cannot be made.",
+                value: "The owner of %@ hasn’t set it up properly and a secure connection can’t be created.",
```

Comment:
since we are changing the string, lets move the old one to the old struct. See process here:
https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#how-to-modify-a-string

Also, since we have %@, lets add a comment to clarify what that value represents

Reply:
Thank you for pointing this out!
"On error page, this is the description for generic error. The placeholder will be replaced by the site url". 
Does this work?

Reply:
works for me! thanks!~